scripts/dapp/webpack.config.js

```diff
@@ -41,20 +38,19 @@ const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
  *                                                          will be increased drastically
  * @return     {any}            webpack config
  */
-module.exports = function(
+module.exports = function getWebpackConfig(
   name,
   dist,
   transpileOnly = false,
-  prodMode = false,
-  externals = getExternals()
+  _prodMode = false,
```

Comment:
I think this would also work, so you don't need an extra variable for it.
 
```suggestion
  prodMode = process.env.NODE_ENV === 'production',
```